core/src/main/scala/cromwell/core/io/AsyncIoActorClient.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell.core.io
+
+import akka.actor.ActorRef
+import cromwell.core.path.BetterFileMethods.OpenOptions
+import cromwell.core.path.Path
+
+import scala.concurrent.Future
+
+/**
+  * Helper trait to used async Io method easily
+  */
+trait AsyncIoActorClient {
```

Comment:
ToL Consider making `asyncIO` public, thus avoiding the need to write the wrapper methods.